istioctl/cmd/kubeinject.go

```diff
@@ -91,7 +91,7 @@ func (e ExternalInjector) Inject(pod *corev1.Pod, deploymentNS string) ([]byte,
 			return nil, err
 		}
 	}
-	tlsClientConfig := &tls.Config{RootCAs: certPool}
+	tlsClientConfig := &tls.Config{RootCAs: certPool, MinVersion: tls.VersionTLS12}
```

Comment:
Is explicitly setting a tls version required? Is there anything wrong using the default tls version setting?